recipe/build.sh

```diff
@@ -2,10 +2,29 @@
 
 set -ex
 
+mkdir build
+cd build
+Python3_NumPy_INCLUDE_DIR="$(python -c 'import numpy;print(numpy.get_include())')"
+Python3_NumPy_INCLUDE_DIRS="$(python -c 'import numpy;print(numpy.get_include())')"
+
+Python_NumPy_INCLUDE_DIR="$(python -c 'import numpy;print(numpy.get_include())')"
+Python_NumPy_INCLUDE_DIRS="$(python -c 'import numpy;print(numpy.get_include())')"
+cmake ${CMAKE_ARGS}                                                \
+    -DPython3_NumPy_INCLUDE_DIRS:PATH=DPython3_NumPy_INCLUDE_DIRS  \
+    -DPython3_NumPy_INCLUDE_DIR:PATH=DPython3_NumPy_INCLUDE_DIR    \
+    -DPython_NumPy_INCLUDE_DIRS:PATH=DPython_NumPy_INCLUDE_DIRS    \
+    -DPython_NumPy_INCLUDE_DIR:PATH=DPython_NumPy_INCLUDE_DIR      \
```

Comment:
```suggestion
    -DPython3_NumPy_INCLUDE_DIRS:PATH=$Python3_NumPy_INCLUDE_DIRS  \
    -DPython3_NumPy_INCLUDE_DIR:PATH=$Python3_NumPy_INCLUDE_DIR    \
    -DPython_NumPy_INCLUDE_DIRS:PATH=$Python_NumPy_INCLUDE_DIRS    \
    -DPython_NumPy_INCLUDE_DIR:PATH=$Python_NumPy_INCLUDE_DIR      \
```


Is this a typo or I am missing something?

Reply:
thanks.... sorry about  that

Reply:
wow you have revived my thoughts....